src/App.js

```diff
@@ -28,22 +29,29 @@ const alex = {
 
 const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
-    </section>
+    <Person
+      name={misha.name}
```

Comment:
If you person will accept some object in props, you can use
`<Person  person={misha} />`...
 
 see what I mean below